src/site/src/content/interventions.mdx

```diff
@@ -35,16 +35,26 @@ The intervention reduced demographic parity difference only slightly, from 0.193
 
 #### Equalised odds
 
-The reduction in equalised odds is more significant, reducting equalised odds difference from 0.128 to 0.074.
+The reduction in equalised odds is more significant, reducting equalised odds difference from 0.128 to 0.074. Further details on mean scores are displayed in two graphs below which compare FTU and baseline.
+
+<PlotLoader source={"finance/interventions/ftu-bl-eo"} />
 
 <PlotLoader source={"finance/interventions/ftu-eo"} />
 
 #### Equal opportunity
 
-The reduction in equal opportunity is identival to equalised odds.
+The reduction in equal opportunity is identical to equalised odds.
 
 <PlotLoader source={"finance/interventions/ftu-eopp"} />
 
+#### Calibration
```

Comment:
can we swap out FTU for "fairness through unawareness"? feels a little too informal

Reply:
Yes of course